isympy.py

```diff
@@ -1,8 +1,5 @@
-# XXX: Don't put a newline here, or it will add an extra line with
-# isympy --help
-#  |
-#  v
-"""Python shell for SymPy.
+"""
```

Comment:
So the XXX doesn't apply?

Other than this, I don't have any questions...but also no experience. But if you use this and nobody else is able to review, I trust your judgement. Is this ready to go?